<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="commit_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit changes?</string>
   </property>
  </widget>
  <widget class="QListWidget" name="commit_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="commit_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
